.gitignore

```diff
@@ -44,3 +44,10 @@ test-encode-task.json
 docs/model_results/*
 model_results/*
 docs/model_results.json
+speed_test.py
+
+# fasttext model
+*.bin
```

Comment:
Yeah makes a lot of sense I just always manually `git add` everything, sorry for that

Reply:
No problem